app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should verify that the input string contains exactly three elements separated by spaces, and that the first element is 'cp'. This is necessary to ensure the command is valid according to the task requirements.